### DISCUSSION TITLE
[Feature Suggestion] Profile Portfolio

### DISCUSSION THREAD

**Discussion by sethsandaru** [community/community#62132] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello team,

I think it would be nice if our GitHub profile would have a new "Portfolio" tab. 

So we can add our featured works there (up to 20 items maybe?). 

=> Much better than 6 pinned items on our main profile pages.

Thanks

**Comment by iSemary** [comment]
Great idea, it would help developers to show up their works!

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @sethsandaru, we're happy you're here! You are more likely to get a useful response if you are posting your feedback in the applicable category. I've gone ahead and moved it for you. Good luck!
